pkg/deploy/generator/resources_rp.go

```diff
@@ -8,8 +8,8 @@ import (
 	"fmt"
 	"strings"
 
+	sdkcosmos "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/cosmos/armcosmos/v2"
 	mgmtcompute "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-06-01/compute"
-	mgmtdocumentdb "github.com/Azure/azure-sdk-for-go/services/cosmos-db/mgmt/2021-01-15/documentdb"
```

Comment:
We are still using this "old" library in 2 other places. Do we need to change those too?

Reply:
The reason for using sdk track 2 here is because the required parameters for this change isn't available in sdk track 1. I suppose, changes in rest of the places would be better done in a seperate PR that would be specifically for migration from sdk track 1 to sdk track 2.

Reply:
We still need to do track 2 migration in several places. I know Dan was tyring to track (ha pun) that down a month or so ago but I haven't had a chance to look deeply into it.

Reply:
There was this [PR](https://github.com/Azure/ARO-RP/pull/3255) which migrated documentdb client to track 2 sdk, it didn't cover this part though. A separate PR for that would be a better option.